### PR TITLE
- use Routers;

### PR DESCRIPTION
**Pull request by dg1886**
- install Redux, React-redux, redux-saga dependencies;

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,12 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.5",
+    "react-router-dom": "^5.3.0",
     "react-scripts": "4.0.3",
+    "redux": "^4.1.1",
+    "redux-saga": "^1.1.3",
+    "saga": "^4.0.0-alpha",
```
Comment (Nehved) [inline package.json]: remove

File: src/Containers/Auth/index.jsx [hunk 2]
```diff
@@ -8,7 +9,7 @@ import {
 
 const types = ['Log In', 'Registration'];
 
-const AuthPage = () => {
+const AuthPage = ({ setAuthDone }) => {
```
Comment (Nehved) [inline src/Containers/Auth/index.jsx]: delete if not needed

File: package.json (hunk 1 above)
Comment (dg1886) [reply package.json]: done

File: src/Containers/Auth/index.jsx (hunk 2 above)
Comment (dg1886) [reply src/Containers/Auth/index.jsx]: done

**Pull request merged**